### PR TITLE
Links added to footer

### PR DESCRIPTION
**Pull request by AceDheGeneral**
footer

### CODE REVIEW

File: apps/client/app/components/footer/index.tsx [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
   TWITTER_ACCOUNT_URL,
 } from '@/constants/index.ts'
-import {type JSX} from 'react/jsx-runtime'
+import { type JSX } from 'react/jsx-runtime'
+import { Link } from '@remix-run/react'
+
+const insideNavigation = [
+  { name: 'Pricing', href: '/#pricing', routeType: 'href' },
+  { name: 'Terms Of Use', href: '/terms', routeType: 'link' },
+]
```
Comment (Bendomey) [inline apps/client/app/components/footer/index.tsx:13]: I believe using the `Link` component should work for both scenarios.

**Review by arkhurst — COMMENTED** [review]
looks good! 
going forward can we add the URL to the task on trello

**Comment by AceDheGeneral** [comment]
> looks good! going forward can we add the URL to the task on trello

Well Noted

**Comment by AceDheGeneral** [comment]
sorry about the commit name

File: apps/client/app/components/footer/index.tsx [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
   TWITTER_ACCOUNT_URL,
 } from '@/constants/index.ts'
-import {type JSX} from 'react/jsx-runtime'
+import { type JSX } from 'react/jsx-runtime'
+import { Link } from '@remix-run/react'
+
+const insideNavigation = [
+  { name: 'Pricing', href: '/#pricing', routeType: 'link' },
+  { name: 'Terms Of Use', href: '/terms', routeType: 'link' },
+]
```
Comment (Bendomey) [inline apps/client/app/components/footer/index.tsx:13]:
```suggestion
  { name: 'Pricing', href: '/#pricing' },
  { name: 'Terms Of Use', href: '/terms' },
]
```

I don't think you need this anymore, right ?🤔

File: apps/client/app/components/footer/index.tsx [hunk 3]
```diff
@@ -91,7 +98,28 @@ export const Footer = () => {
...
+                  </Fragment>
+                )
+              })
+            }
+
+          </div>
+
         </div>
+
+
```
Comment (Bendomey) [inline apps/client/app/components/footer/index.tsx:122]:
```suggestion
```

File: apps/client/app/components/footer/index.tsx [hunk 4]
```diff
@@ -91,6 +98,25 @@ export const Footer = () => {
...
+                      >
+                        {item.name}
+                      </Link>
+                  </Fragment>
+                )
+              })
+            }
+
+          </div>
+
```
Comment (Bendomey) [inline apps/client/app/components/footer/index.tsx:119]:
```suggestion
```

File: apps/client/app/components/footer/index.tsx [hunk 5]
```diff
@@ -91,6 +98,24 @@ export const Footer = () => {
...
+                        to={item.href}
+                        className="text-sm font-semibold leading-6 text-gray-400 hover:text-gray-500"
+                      >
+                        {item.name}
+                      </Link>
+                  </Fragment>
+                )
+              })
+            }
+
```
Comment (Bendomey) [inline apps/client/app/components/footer/index.tsx:117]:
```suggestion
```

**Review by Bendomey — APPROVED** [review]

**Pull request merged**